src/components/NavBar.tsx

```diff
@@ -1,15 +1,31 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
-import '../styles/NavBar.scss'
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+import '../styles/BurgerButton.scss'
+import { useState } from 'react';
+import '../styles/NavBar.scss';
+
 
 export function NavBar() {
+    const [closeBurger, setBurgerState] = useState(true);
+
     return (
-        <div className='nav-bar'>
-            <Link className='nav-link' to='/'> Home </Link>
-            <Link className='nav-link' to='/spirit' >Spirit </Link>
-            <Link className='nav-link' to='/opportunity'>Opportunity</Link>
-            <Link className='nav-link' to='/curiosity'>Curiosity</Link>
-            <Link className='nav-link' to='/timeline'> Timeline</Link>
+        <div className='header'>
+            <div className={closeBurger ? 'nav-bar' : 'nav-bar-mobile'} data-testid="nav-bar">
+                <Link to='/' className = 'nav-link'> Home </Link>
+                <Link to='/spirit' className = 'nav-link'>Spirit </Link>
+                <Link to='/opportunity'className = 'nav-link'>Opportunity</Link>
+                <Link to='/curiosity'className = 'nav-link'>Curiosity</Link>
+                <Link to='/timeline'className = 'nav-link' > Timeline</Link>
+            </div>
+
+            <div className="burger-button" 
+                onClick={() => {setBurgerState(!closeBurger)}
+            } data-testid= 'burger-test'>
+                <div className={closeBurger ? "opened-bar-1" : "closed-bar-1"} 
```

Comment:
is there a reason you decided to not have the open close on the parent element?

Reply:
We couldn't get the other way to work - burger button disappeared :(